pkg/eks/nodegroup.go

```diff
@@ -128,59 +126,3 @@ func (c *ClusterProvider) WaitForNodes(clientSet *clientset.Clientset, ng *api.N
 
 	return nil
 }
-
-// ValidateConfigForExistingNodeGroups looks at each of the existing nodegroups and
```

Comment:
This was introduced as a temporary fix to what seemed like DNS-only issue (#414), clearly a little too involved and I think we have a better answer now.